tests/NoFutureImportTest.py

```diff
@@ -25,3 +26,24 @@ def test_invalid(self):
         self.check_invalidity(self.uut,
                               ['from __future__ import division, '
                                'print_function'])
+
+    def test_multiline_imports(self):
+        file_text = ['from __future__ import with_statement, \\\n',
+                     '                       print_function, \\\n',
+                     '                       generators\n']
+
+        self.check_invalidity(self.uut, file_text)
+
+    def test_compound_imports(self):
+        file_text = ['from __future__ import unicode_literals;',
+                     'from __future__ import nested_scopes; x = 2\n']
+
+        self.check_invalidity(self.uut, file_text)
+
+    def test_multiline_compound_imports(self):
+        file_text = ['from __future__ import nested_scopes, \\\n',
+                     '                       generators;',
+                     'from __future__ import absolute_import;',
+                     'x = 2\n']
+
+        self.check_invalidity(self.uut, file_text)
```

Comment:
`check_invalidity` is not good enough to assert that the correct transformation occurred.